cluster/gce/trusty/configure-helper.sh

```diff
@@ -726,7 +726,7 @@ Note: This looks like a development version, which might not be present on GitHu
 If it isn't, the closest tag is at:
   https://github.com/kubernetes/kubernetes/tree/${gitref}
 "
-    gitref="${version//*+/}"
+    gitref=$(echo ${version} | sed -e 's/.*+//')
```

Comment:
Why is this needed? And why the change versus `head` (where it looks like the left in both gci and trusty)?

```
$ version=1.4.0-alpha.0.210+6209b1b60cd0d9
$ echo "${version//*+/}"
6209b1b60cd0d9
$ echo $(echo ${version} | sed -e 's/.*+//')
6209b1b60cd0d9
```